velox/functions/lib/Re2Functions.cpp

```diff
@@ -30,6 +33,8 @@ using ::facebook::velox::exec::VectorFunction;
 using ::facebook::velox::exec::VectorFunctionArg;
 using ::re2::RE2;
 
+static const int kMaxCompiledRegexes = 20;
```

Comment:
I wonder where the 20 comes from?
Maybe add a comment, like most times it is 20 or we have 16 threads or something else...

Reply:
So what is the consequences of this limit?
What if we have 100 rows and each one of them results in a different regex, because if takes, say, pattern from a column.
Will such query fail?

Reply:
Yes, a query that exceeds the limit of unique regexes will fail. This limit is to protect the cluster. Regex is expensive to compute and compiling regex for every row can easily bring down a machine.